generators/client-2/templates/src/main/webapp/app/account/password-reset/finish/_password-reset-finish.component.ts

```diff
@@ -0,0 +1,54 @@
+import {Component, OnInit, Inject, Renderer, ElementRef} from '@angular/core';
+import { TranslatePipe } from '../../../shared/translate.pipe';
+
+@Component({
+    selector: 'password-reset-finish',
+    templateUrl: 'app/account/password-reset/finish/password-reset-finish.html'/*,
+    pipes: [TranslatePipe]*/
```

Comment:
you might be on an older version of base code which was on rc4. Try to rebase to current angular2 branch,  else it should be ok when I merge this. so you can remove this line